CONTRIBUTING.md

```diff
@@ -484,7 +484,9 @@ The tests require a few environment variables to be set:
 when deploying Concourse in the k8s cluster
 - `CONCOURSE_IMAGE_NAME`: the name of the image to use when deploying Concourse
 to the Kubernetes cluster
-- `CHARTS_DIR`: location in the filesystem where a copy of [`the Concourse Helm
+- `HELM_CHARTS_DIR`: path to a clone of the [`helm/charts`][helm-charts] repo. This is used
+to define the postgres chart that Concourse depends on.
```

Comment:
so, I mention postgres here but I think k8s-topgun actually depends on multiple other charts. Is that true? @cirocosta @taylorsilva

Reply:
yeah! that's correct - there's `prometheus` too (as you probably know at this point :sweat_smile: )